src/Microsoft.Identity.Web.TokenAcquisition.Abstractions/ApplicationOptions/MicrosoftAuthenticationOptions.cs

```diff
@@ -76,7 +78,9 @@ internal bool IsB2C
         {
             get => !string.IsNullOrWhiteSpace(DefaultUserFlow);
         }
+        #endregion
```

Comment:
I don't think we need the regions necessarily, but if so, please add the region name at the end of the region : ex `#endregion B2C`